Runtime/Tabs/DynamicTab.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using UnityEngine;
+
+namespace Innoactive.Creator.Core.Tabs
+{
+    internal class DynamicTab : ITab
```

Comment:
Yes i know this is internal, but I have no clue how to use this, would love to see some information about this.